README.md

```diff
@@ -213,13 +212,24 @@ The following optional query arguments are available:
     ([`--time`](https://slurm.schedmd.com/sbatch.html#OPT_time))
 
 - Jupyter(Lab) configuration:
+
   - `default_url`: The URL to open the Jupyter environment with: use `/lab` to
     start [JupyterLab](https://jupyterlab.readthedocs.io/en/stable/) or use
     [JupyterLab URLs](https://jupyterlab.readthedocs.io/en/stable/user/urls.html)
-  - `environment_path`: Path to Python environment bin/ used to start Jupyter
+  - `environment_id`: Name of the Python environment defined in the
+    configuration used to start Jupyter
+  - `environment_path`: Path to the Python environment bin/ used to start
+    Jupyter
+  - `environment_modules`: Space-separated list of
+    [environment module](https://modules.sourceforge.net/) names to load before
+    starting Jupyter
   - `root_dir`: The path of the "root" folder browsable from Jupyter(Lab)
     (user's home directory if not provided)
 
+  Note: To use a Jupyter environment defined in the configuration, only provide
+  its `environment_id`. To use a custom Jupyter environment, provide the
+  corresponding `environment_path` and/or `environment_modules`.
```

Comment:
I hope this is clear enough. It starts to be complicated with having different options...

Reply:
You could give examples URL to ease the understanding

Reply:
Yes, good idea, done!